src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        String randomColor = ColorSupplier.getRandomColor();
+
+        Ball ball = new Ball();
+        ball.color = randomColor;
+        ball.number = randomNumber;
+
+        return ball;
+    }
+}
```

Comment:
The class 'Lottery' should have an access modifier, preferably 'public' if it's meant to be used outside of its package.